fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/database/DatabasePasswordEncryptor.java

```diff
@@ -37,6 +38,20 @@ public class DatabasePasswordEncryptor implements PasswordEncryptor {
 
     private final FineractProperties fineractProperties;
 
+    //CHECKSTYLE.OFF: IllegalCatch - Turn off checkstyle for main method
```

Comment:
You might wanna use this `@SuppressWarnings("checkstyle:regexpsinglelinejava")`

Reply:
Thanks.